sandy/njoy.py

```diff
@@ -729,6 +731,55 @@ def _groupr_input(endfin, pendfin, pendfout, mat,
     `str`
         groupr input text
 
+    Examples
+    --------
+    Default test without keyword arguments
+    >>> print(sandy.njoy._groupr_input(20, 21, 22, 9237))
+    groupr
+    20 21 0 22 /
+    9237 2 0 2 0 1 1 0 /
+    /
+    293.6/
+    10000000000.0/
+    3/
+    0/
+    0/
+
+    Test argument `temp`
+    >>> print(sandy.njoy._groupr_input(20, 21, 22, 9440, temp=600))
+    groupr
+    20 21 0 22 /
+    9440 2 0 2 0 1 1 0 /
+    /
+    600.0/
+    10000000000.0/
+    3/
+    0/
+    0/
+
+    Test argument `iwt`
+    >>> print(sandy.njoy._groupr_input(20, 21, 22, 9237, iwt=6))
+    groupr
+    20 21 0 22 /
+    9237 2 0 6 0 1 1 0 /
+    /
+    293.6/
+    10000000000.0/
+    3/
+    0/
+    0/
+
+    Test argument `ign`
```

Comment:
In theory there should also be an option `ek` as for errorr, when `ign==1`, right?!